charts/pulsar/templates/broker-service.yaml

```diff
@@ -43,7 +44,17 @@ spec:
   - name: "{{ .Values.tlsPrefix }}pulsarssl"
     port: {{ .Values.broker.ports.pulsarssl }}
   {{- end }}
-  clusterIP: None
+  {{- if and .Values.broker.service .Values.broker.service.clusterIP }}
+  {{- if eq .Values.broker.service.clusterIP "auto" }}
```

Comment:
Why not use an empty value for this purpose and have `None` as the default in values.yaml?
Simply just putting the clusterIP value from values.yaml to the template and not doing anything special.

Reply:
Cannot find a good motivation, atm. Makes sense to me. 

Addressed by 4d8e283